api/docs/http_catalog.md

```diff
@@ -42,7 +42,7 @@ Three parameters should be provided to the POST or GET:
 
 #### Catalog found
 
-    POST /env/catalog/elmo.mydomain.com
+    POST /puppet/v3/catalog/elmo.mydomain.com?environment=:environment
```

Comment:
Whoops, should have had that be `?environment=env`. I was just trying to keep all the environments/urls the same, just updated for the new structure.